### ISSUE TITLE
.jar File Prompt "no main manifest attribute"

### ISSUE THREAD

**Issue by LeopoldChou** [Earthcomputer/EnchantmentCracker#352]
Env: macOS 10.15, java 18
Prompt "**no main manifest attribute**" when running _enchcracker-1.6.jar_ file
<img width="777" alt="截屏2023-02-23 下午2 28 22" src="https://user-images.githubusercontent.com/68420848/220835370-c04b9666-c183-47fb-b68a-909a6272045f.png">

**Comment by Earthcomputer** [comment]
Yeah you're supposed to launch it with the bash script, not directly with the jar
